blueocean-dashboard/src/main/js/components/testing/TestService.js

```diff
@@ -3,6 +3,18 @@ import TestLogService from './TestLogService';
 
 const PAGE_SIZE = 100;
 
+// Source: https://stackoverflow.com/questions/286141/remove-blank-attributes-from-an-object-in-javascript/38340730#38340730
+const stripNullAndUndefined = obj =>
```

Comment:
If you end up needing this, don't do it like this :) Just `obj => JSON.parse(JSON.stringify(obj))` is easier might be faster, and instead of hanging up will throw if a cycle is detected.

Also IMO (I'm not signing the cheques so feel free to ignore, just a style nitpick) top level functions should only be in arrow form if they're React components, and then only to make it clear they're not constructors.

Reply:
Not needed now anyways. Done